### DISCUSSION TITLE
能否添加代码补全功能，以及拓展功能

### DISCUSSION THREAD

**Discussion by xgugugu** [cpeditor/cpeditor-cn#13] [想法]
如果在敲代码时能有自动补全，那就更好了
另外，拓展功能应该是一个强大的IDE所必须有的

**Comment by ouuan** [comment]
代码补全: https://github.com/cpeditor/cpeditor/issues/156

插件可以参考 https://github.com/cpeditor/cpeditor/issues/916 和 https://github.com/cpeditor/cpeditor/issues/914
